soap-authorization-policy/src/main/apiman/policyDefs/schemas/soap-authorization-PolicyDef.schema

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "SOAP Authorization Policy Configuration",
+  "type": "object",
+  "options": {
+    "disable_collapse": true
+  },
+  "properties": {
+    "rules": {
+      "title": "Add Authorization Rule",
+      "description": "Manage the list of fine-grained authorization rules in the box below. If you want a single required role to protect your entire API, simply add one item with a path of \".*\" and an action of \"*\".",
+      "type": "array",
+      "format": "table",
+      "items": {
+        "type": "object",
+        "title": "Rule",
+        "properties": {
+          "pathPattern": {
+            "title": "Ignored URI",
+            "type": "string"
+          },
+          "soapAction": {
+            "title": "SOAPAction",
+            "type": "string"
+          },
+          "role": {
+            "title": "Role",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "multiMatch": {
+      "title": "Multiple Match Action",
+      "type": "string",
+      "enum": [
+        "I want the request to pass when all matching authorization rules pass.",
+        "I want the request to pass when at least one matching authorization rules pass."
```

Comment:
The reason I did it this way is because of the formatting in the UI. For example, we would need to split it into three parts:

1. "I want the request to pass when "
2. ["all", "at least one"]
3. " matching authorization rules pass."

Each property is automatically displayed on a new line unless it is something like an array, but we'd want all three parts to be on the same line as it is for the Authorization Policy.

It is difficult to use CSS to ensure that all three values are on the same line because forms are not assigned unique classes or id's, but what I can do is see if we can use the grid layout instead.